ovs_common_uapi.rs is not built correctly with `make gen-bindings`

when I directly run `make gen-bindings` on my laptop against `main`, I get the following error message and `retis/src/bindings/ovs_common_uapi.rs` is not built correctly:
```
[BINDINGS]   processing /home/akaris/development/retis/retis/src/module/ovs/bpf/include/ovs_common.h ...
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:86:90: error: unknown type name '__u64'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:110:49: error: unknown type name '__u32'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:122:23: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:122:8: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:122:14: error: function cannot return function type 'int (void)'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:122:8: error: illegal initializer (only variables can be initialized)
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:185:57: error: unknown type name '__u32'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:201:23: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:201:8: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:201:14: error: function cannot return function type 'int (void)'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:201:8: error: illegal initializer (only variables can be initialized)
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:214:38: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:214:23: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:214:29: error: function cannot return function type 'int (void)'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:214:23: error: illegal initializer (only variables can be initialized)
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:235:66: error: unknown type name '__u32'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:235:98: error: unknown type name '__u32'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:235:109: error: unknown type name '__u64'
/home/akaris/development/retis/retis/src/.out/bpf/bpf_helper_defs.h:264:66: error: unknown type name '__u32'
fatal error: too many errors emitted, stopping now [-ferror-limit=]
```
I hit the same on my branch, so after generating the bindings, I simply stashed the changes to `ovs_common_uapi.rs`. It's probably a dependency issue on my side, but it not, something worth investigating / fixing?

--- a/retis/src/bindings/ovs_common_uapi.rs
+++ b/retis/src/bindings/ovs_common_uapi.rs
@@ -7,11 +7,6 @@ pub type u64_ = __u64;
 pub type bool_ = bool;
 #[repr(C)]
 #[derive(Debug, Default, Copy, Clone)]
-pub struct ovs_config {
-    pub action_max: u32_,
-}
-#[repr(C)]
-#[derive(Debug, Default, Copy, Clone)]
 pub struct upcall_context {
     pub ts: u64_,
     pub cpu: u32_,

--- a/retis/src/module/ovs/bpf/include/ovs_common.h
+++ b/retis/src/module/ovs/bpf/include/ovs_common.h
@@ -1,6 +1,7 @@
 #ifndef __MODULE_OVS_COMMON__
 #define __MODULE_OVS_COMMON__
 
+#include <vmlinux.h>
 #include <bpf/bpf_core_read.h>
 
 #include <common_defs.h>

